DuckDuckGo/AppDelegate.swift

```diff
@@ -33,6 +33,7 @@ import Configuration
 import Networking
 import DDGSync
 import SyncDataProviders
+import Subscription
```

Comment:
Question, shouldn't we guard those `import Subscription` in the `#if SUBSCRIPTION` checks?
This way we would ensure that the imports are not satisfied in the targets lacking the `SUBSCRIPTION` flag and we would ensure on compile time that no code using it would  be in those builds.